apps/ensnode/src/plugins/base.eth/handlers/Registrar.ts

```diff
@@ -1,11 +1,18 @@
 import { ponder } from "ponder:registry";
 import schema from "ponder:schema";
-import { makeSubnodeNamehash, tokenIdToLabel } from "ensnode-utils/subname-helpers";
+import { makeSubnodeNamehash, uint256ToHex32 } from "ensnode-utils/subname-helpers";
+import type { Labelhash } from "ensnode-utils/types";
 import { zeroAddress } from "viem";
 import { makeRegistrarHandlers } from "../../../handlers/Registrar";
 import { upsertAccount } from "../../../lib/db-helpers";
 import { ownedName, pluginNamespace } from "../ponder.config";
 
+/**
+ * Base RegistrarController contract's tokenId is uint256(labelhash)
+ * https://github.com/base-org/basenames/blob/main/src/L2/RegistrarController.sol#L488
+ */
+export const tokenIdToLabelhash = (tokenId: bigint): Labelhash => uint256ToHex32(tokenId);
```

Comment:
```suggestion
export const tokenIdToDirectSubnameLabelhash = (tokenId: bigint): Labelhash => uint256ToHex32(tokenId);
```

Appreciate that's a mouthful.. But putting priority on clarity and avoiding misinterpretations.